docs/index.rst

```diff
@@ -84,20 +84,21 @@ Parcels depends on a working Python installation, a netCDF installation, a C
 compiler, and various Python packages.
 
 The simplest way to install Parcels is to use Anaconda and Conda-Forge for all
-requirements. This is the “batteries-included” solution probably suitable for most users.
+requirements. This is the “batteries-included” solution probably suitable for most users. 
+Note that this works best if Python is not installed yet. Otherwise, see the instructions at the bottom of this section.
```

Comment:
I'd go for a formulation that cannot be confused with the operating system's Python installation present on virtually all Linux machines and most of the Macs.  What about:

> Note that if you already have an existing Python setup that you wish to use for Parcels as well, you should follow the instructions at the bottom of this section.